File: python/roman_numerals_spec.py
from roman_numerals import to_roman

print(to_roman(1) == 'I')
print(to_roman(3) == 'III')
print(to_roman(4) == 'IV')
# add tests to cover different edge cases

print(to_roman(49));     #XLIX
print(to_roman(99));     #XCIX
print(to_roman(499));    #CDXCIX
print(to_roman(944));    #CMXLIV
print(to_roman(999));    #CMXCIX
print(to_roman(150));    #CL

File: js/romanNumeralsSpec.js
var rn = require("./romanNumerals");

// console.log(rn.toRoman(1) === 'I');
// console.log(rn.toRoman(3) === 'III');
// console.log(rn.toRoman(4) === 'IV');

// console.log(rn.toRoman(4) === 'IIII');
// console.log(rn.toRoman(9) === 'VIIII');
// console.log(rn.toRoman(52) === 'LII');
//  console.log(rn.toRoman(4));      //IV
//  console.log(rn.toRoman(9));      //IX
// console.log(rn.toRoman(40));     //XL
console.log(rn.toRoman(49));     //XLIX
console.log(rn.toRoman(99));     //XCIX
console.log(rn.toRoman(499));    //CDXCIX
console.log(rn.toRoman(944));    //CMXLIV
console.log(rn.toRoman(999));    //CMXCIX
console.log(rn.toRoman(150));    //CL

// console.log(rn.toRoman(4005));    //MMMMV
// console.log(rn.toRoman(4949));    //MMMMCMXLIX
// console.log(rn.toRoman(949));    //CMXLIX



